File: stress/t/flt_combo.c
#include <stdio.h>
#include <math.h>
int main(void){
  double x,y,d,a,b,r;
  x=3.0; y=4.0; d=sqrt(x*x+y*y);
  printf("d= %f\n",d);
  a=exp(log(7.5));
  printf("a= %f\n",a);
  b=pow(2.0,0.5)*pow(2.0,0.5);
  printf("b= %f\n",b);
  r=fmax(sin(1.0),cos(1.0));
  printf("r= %f\n",r);
  r=fabs(-(sqrt(2.0)));
  printf("r= %f\n",r);
  return 0;
}

File: stress/t/flt_combo.f
      PROGRAM FCOMBO
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      X=3.0D0
      Y=4.0D0
      D=SQRT(X*X+Y*Y)
      WRITE(*,'(A,F0.6)') 'd= ',D
      A=EXP(LOG(7.5D0))
      WRITE(*,'(A,F0.6)') 'a= ',A
      B=(2.0D0**0.5D0)*(2.0D0**0.5D0)
      WRITE(*,'(A,F0.6)') 'b= ',B
      R=MAX(SIN(1.0D0),COS(1.0D0))
      WRITE(*,'(A,F0.6)') 'r= ',R
      R=ABS(-(SQRT(2.0D0)))
      WRITE(*,'(A,F0.6)') 'r= ',R
      END
